### DISCUSSION TITLE
Embed workflow designer in my application

### DISCUSSION THREAD

**Discussion by maelmahdy86** [elsa-workflows/elsa-core#3268] [Q&A]
Hi,

First of all, I'd like to thank you for this great library and its awesome features & capabilities. It's very powerful and promising

Now, I've a solution that supports multi-tenancy and a lot of security restrictions over data and workflows so current dashboard won't be enough for us so I thought we can build our own dashboard and apply all our security checks on it but we need to embed the workflow designer component in the application

Is it support to use the workflow designer as a standalone embeddable component?

**Comment by eros404** [comment]
You can do that to display the workflow editor :

```cshtml
<elsa-studio-root server-url="server-url" monaco-lib-path="_content/Elsa.Designer.Components.Web/monaco-editor/min">
         <elsa-workflow-definition-editor-screen workflow-definition-id="workflow-definition-id" />
</elsa-studio-root>
```

**Reply by taurisjoe** [reply]
Is there a way to resize the elsa-studio-root element?  I've embedded it into a Blazor Server application but I've been unsuccessful in changing the height so it doesn't spill outside the window.  It's annoying scrolling down to get to the publish button.

I've tried creating a style tag and applying a class to a div tag surrounding it, but it just ignores it.
